object_detection/yolov3_tensorflow/readme.md

```diff
@@ -0,0 +1,96 @@
+# Tensorflow Implementation of YOLOv3
+
+Download weights in the repo containing files
+```bash 
+$ wget https://pjreddie.com/media/files/yolov3.weights
+```
+## Usage
+```bash
+$ python3 main.py 
+```
+In order to use the model on custom images add them into images directory
+
+> **_NOTE:_** on Colab Notebook use following command:
+```python
+!git clone https://github.com/antreev-brar/model-zoo.git
+%cd /content/model-zoo/yolov3
+!wget https://pjreddie.com/media/files/yolov3.weights
+!python main.py
+```
```

Comment:
You may define hyperparameters in the form of parser and copy the output of `python main.py --help` here. This will make your code more flexible.